README.md

```diff
@@ -47,6 +47,43 @@ environment variables can be used to determine environment).  For example in
     production:
       "foo,bar,baz": 4
 
+### YAML file config [UPDATE]
+The `config/resque-pool.yml` can recognise whether workers of a queue should
+be forking or non-forking. For that reason the declaration of worker count
+has changed and it should follow the template below:
+
+```
+    foo:
+      - workers: 1
+    production:
+      "bar,baz":
+        - workers: 2
+      "koko":
+        - workers: 4
+        - fork: false
```

Comment:
We could follow Resque's conventions and use `fork_per_job` here too.

Reply:
Fixed!